alembic/versions/51e4adb4b60_add_watch_list.py

```diff
@@ -28,6 +28,15 @@ def upgrade():
         sa.Column('action', sa.UnicodeText(), nullable=False),
     )
 
```

Comment:
Oh neat, I was not aware that's what we were doing. That's definitely something to keep in mind.